cg/meta/demultiplex/housekeeper_storage_functions.py

```diff
@@ -92,6 +95,20 @@ def add_demux_logs_to_housekeeper(
             LOG.error(f"Cannot find demux log file {log_file_path}. Error: {e}.")
 
 
+def add_run_parameters_file_to_housekeeper(
+    flow_cell_name: str, flow_cell_run_dir: Path, hk_api: HousekeeperAPI
+) -> None:
+    """Add run parameters file to Housekeeper."""
+    flow_cell_path = Path(flow_cell_run_dir, flow_cell_name)
+    flow_cell = FlowCellDirectoryData(flow_cell_path)
+    run_parameters_file_path: Path = flow_cell.run_parameters_path
+    tag_names: list[str] = [SequencingFileTag.RUN_PARAMETERS, flow_cell.id]
+    hk_api.add_file_to_bundle_if_non_existent(
+        file_path=run_parameters_file_path, bundle_name=flow_cell.id, tag_names=tag_names
+    )
+    LOG.info(f"Added run parameters file {run_parameters_file_path} to Housekeeper.")
```

Comment:
Maybe a bit over the top with other logs but would it make sense to add the bundle here? For example
```suggestion
    LOG.info(f"Added run parameters file {run_parameters_file_path} to {flow_cell.id} in Housekeeper.")
```

Reply:
I will add it and see how it looks!

Reply:
looks good, I will leave it